packages/contracts/src/contracts/current/protocol/Exchange/libs/LibOrder.sol

```diff
@@ -98,6 +98,26 @@ contract LibOrder is
         bytes32 schemaHash = EIP712_ORDER_SCHEMA_HASH;
         bytes32 makerAssetDataHash = keccak256(order.makerAssetData);
         bytes32 takerAssetDataHash = keccak256(order.takerAssetData);
+        // Assembly for more efficiently computing:
+        // keccak256(abi.encodePacked(
```

Comment:
This function does not compute the outer hash, only the inner one (lines 106-119).